styles.css

```diff
@@ -426,7 +471,7 @@ h2 {
   background-size: 100%;
   background-position: bottom;
   position: relative;
-  transition: all 0.5s ease-in-out;
+  transition: all 0.2s ease-in-out;
 }
```

Comment:
- [ ] Nice implementing the design,  but The hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins). In order to match the Figma design let's change the color of your text actually you're using a white color let's change it to ` #FFF5E1;` 🎨👨🏻‍🎨 

-  Adding these features will make your design look great and look exactly like the Figma design, I highly recommend you do this in the meantime, I'll approve this project since the previous reviewer is skipping this.


| Your design | Figma design |
| --- | --- |
| ![Screenshot from 2022-10-11 10-34-07](https://user-images.githubusercontent.com/79058364/195041154-9362481e-7b41-4429-b611-e24a318850c2.png) | ![Screenshot from 2022-10-11 10-38-09](https://user-images.githubusercontent.com/79058364/195041415-387912a7-d126-4878-be96-f9de51e483bc.png)
|

Reply:
### Please ensure you make all required changes and request a re-review